tasks/lib/helpers/git.js

```diff
@@ -52,7 +52,7 @@ function checkout(grunt, dir, branchName, done) {
 
 function merge(grunt, dir, mergeFromBranch, msgPrefix, msg, done) {
 
-  exec('git merge --no-ff -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
+  exec('git merge --no-ff --strategy-option theirs -m "' + msgPrefix + ' ' + msg + '" ' + mergeFromBranch, {
```

Comment:
It's weird, I would also have guessed that this should work. But after the execution of bump I got a merge conflict which had to be resolved manually, but at least with the option "use theirs". Shall I implement it as an option?